src/content/en/2021/capabilities.md

```diff
@@ -1,20 +1,234 @@
+
+#### Write Access
+The Async Clipboard API offers two methods to copy data to the clipboard: The shorthand method [`writeText()`](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText) takes plain text as an argument which the browser then copies to the clipboard. The [`write()`](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/write) method takes an array of clipboard items that could contain arbitrary data. Browsers can decide to only implement certain data formats. The Clipboard API specification specifies a [list of mandatory data types](https://www.w3.org/TR/clipboard-apis/#mandatory-data-types-x) browsers must support as a minimum, including plain text, HTML, URI lists, and PNG images.
+
+```js
+await navigator.clipboard.writeText('hello world');
+
+const blob = new Blob(['hello world'], { type: 'text/plain' });
+await navigator.clipboard.write([
+  new ClipboardItem({
+    [blob.type]: blob,
+  }),
+]);
+```
+
+#### Read Access
+Similar to copying data to the clipboard, there are two methods to paste data back from the clipboard: First, another [shorthand method called `readText()`](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/readText) that returns plain text from the clipboard. Using the [`read()` method](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/read), you access all items in the clipboard in the data formats supported by the browser.
+
+```js
+const item = await navigator.clipboard.readText();
+const items = await navigator.clipboard.read();
+```
+
+The browser may show a permission prompt or a different UI for privacy reasons before granting the website access to the clipboard contents. The Async Clipboard API is available in Chrome, Edge, and Safari. Firefox only supports the `writeText()` method.
+
+With 560,359 (8.91%) desktop and 618,062 (8.25%) mobile sites, the Async Clipboard API (`writeText()` method) is one of the most used Fugu APIs. The `write()` method is used on 1,180 desktop and 1,227 mobile sites. The commercial website [Clipping Magic](https://clippingmagic.com/) allows you to remove the background of an image with the help of an AI algorithm. Just paste an image from the clipboard, and the website will remove its background.
+
+### Web Share API
+The [Web Share API](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share) allows you to share text, a URL, or files from a website or web application with other applications, e.g., mail clients or messengers. To do so, call the [`navigator.share()` method](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share). It takes an object with the data to share with another application. The browser then opens the native share sheet, where the user can select the target application from. The method returns a promise that resolves in case the content was successfully shared; otherwise, it will be rejected.
```

Comment:
```suggestion
The [Web Share API](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share) allows you to share text, a URL, or files from a website or web application with other applications, e.g., mail clients or messengers. To do so, call the [`navigator.share()`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share) method. It takes an object with the data to share with another application. The browser then opens the built-in share sheet, where the user can select the target application from. The method returns a promise that resolves in case the content was successfully shared; otherwise, it will be rejected.
```